Jenkinsfile

```diff
@@ -46,6 +46,11 @@ git merge ${merge_opts} '${commit}' -m '${commit_msg}'
 """)
 }
 
+// https://issues.jenkins-ci.org/browse/JENKINS-37069
+def fix_workspace_label() {
+    sh "chcon -Rt svirt_sandbox_file_t '${env.WORKSPACE}'*"
```

Comment:
I submitted a PR to the docker workflow plugin repository, but for the time being let's use this workaround.